components/p3a/brave_p3a_config.cc

```diff
@@ -0,0 +1,113 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/p3a/brave_p3a_config.h"
+
+#include "base/command_line.h"
+#include "base/logging.h"
+#include "brave/components/p3a/brave_p3a_switches.h"
+#include "brave/components/p3a/buildflags.h"
+
+namespace brave {
+
+namespace {
+
+constexpr uint64_t kDefaultUploadIntervalSeconds = 60;  // 1 minute.
+
+void LoadTimeDelta(base::CommandLine* cmdline,
```

Comment:
All of these `Load*` method names are very confusing imo `MaybeGetTimeDeltaFromCommandLine`. There's also no reason to pass in CommandLine and it should return optional instead of out params

Reply:
returning default values instead, they work fine